app/channel.go

```diff
@@ -2700,7 +2700,14 @@ func (a *App) UpdateSidebarCategories(userId, teamId string, categories []*model
 		return nil, err
 	}
 	message := model.NewWebSocketEvent(model.WEBSOCKET_EVENT_SIDEBAR_CATEGORY_UPDATED, teamId, "", userId, nil)
+
+	var categoryIds = []string
+	for _, category := range categories {
+		categoryIds = append(categoryIds, category.id)
+	}
```

Comment:
Since you are copying an existing slice its better to define the length of the new slice when you make it instead of making a new slice every time you append an element to it

something like 
```suggestion
	categoryIds := make([]string, len(categories))
	for i, category := range categories {
		categoryIds[i] = category.Id
	}
```